src/firefly/js/visualize/ui/MouseReadout.jsx

```diff
@@ -470,12 +255,15 @@ function getSingleMouseReadout(plot, imagePt, toCoordinateName) {
                 break;
 
             case CoordinateSys.PIXEL:
-                var pt = plot.projection.getPixelScaleArcSec();
-                result = `  ${numeral(pt).format(precision3Digit)}"`;
+                obj =readoutItems.pixel;
+                fStr = '0.' + new Array(obj.precision + 1).join('0');
+                result = `${numeral(obj.value).format(fStr)} ${obj.unit || ''}`;
+
```

Comment:
Picky: you could break line 259 and 260 into a simple function as reuse it several times.
_something like this:_
```
const myFormat= (v,precision) => numeral(v).format(padEnd('0.',precision+2,'0') );
```